src/pkg/bundle/dev.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package bundle contains functions for interacting with, managing and deploying UDS packages
+package bundle
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/defenseunicorns/uds-cli/src/config"
+	zarfCLI "github.com/defenseunicorns/zarf/src/cmd"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	zarfUtils "github.com/defenseunicorns/zarf/src/pkg/utils"
+)
+
+// CreateZarfPkg creates a zarf package if its missing when in dev mode
+func (b *Bundle) CreateZarfPkgs(srcDir string) {
+	path := filepath.Join(srcDir, b.cfg.CreateOpts.BundleFile)
+	if err := zarfUtils.ReadYaml(path, &b.bundle); err != nil {
+		message.Fatalf(err, "Failed to read bundle.yaml: %s", err.Error())
+	}
+
+	zarfPackagePattern := `^zarf-.*\.tar\.zst$`
+	for _, pkg := range b.bundle.Packages {
+		if pkg.Repository == "" {
```

Comment:
for readability it might make sense to check the inverse here since we dealing with paths (ie. `if pkg.Path != ""`)

Reply:
so we are just checking to see if its a local zarf package, for the inverse (remote zarf packages) we don't do anything. Added a comment to help make more clear, also let me know if I'm misunderstanding 😃

Reply:
updated to check path instead of repo